### DISCUSSION TITLE
Can't find type file

### DISCUSSION THREAD

**Discussion by birea** [pavlovcik/uad-ui#221] [Q&A] [answered]
Hello. everyone.
I just clone this repo and then run.
But I can't start because this don't have some types files.

contracts/dollar/artifacts/types, contracts/ubiquistick/types and so on.
If you know, let me know
Thanks

**Comment by sergfeldman** [comment]
@leon282 Please, check the Installation in https://github.com/ubiquity/ubiquity-dollar

**Comment by birea** [comment]
![image](https://user-images.githubusercontent.com/87623751/181793272-923e5cbe-9673-4a6a-ac5c-b07f5e133b5b.png)

**Comment by birea** [comment]
I can't find "dollar-types"

**Reply by sergfeldman** [reply]
@Zequez Please take a look and recommend a solution

**Reply by 0x4007** [reply]
Doesn't make sense to me because if it passes CI build then it works. It's not an issue with the codebase. Check your local config settings.

**Comment by 0x4007** [comment, marked as answer]
Make sure that you're cloning from a commit where the build passed. On `development` the latest commit 75185f91210b16d5793a08c3fc73951a3cb36b17 is stable.
